templates/profiles.html

```diff
@@ -197,11 +197,10 @@ <h4 id="name" style="margin:5px"></h4>
             </td>
         </tr>
         <tr>
-            <td style="width:10%"><b>payload:</b></td>
+            <td style="width:10%"><b>payloads:</b></td>
             <td>
-              <select id="ability-payload" class="ability-payload">
-                <option value="" disabled selected>Download a payload before running this ability (optional)</option>
-                {% for p in payloads %}
+              <select id="ability-payload" multiple="multiple">
```

Comment:
2 things:

1) my payloads multi-select box is really thin, like it was hard to tell there were more payloads being displayed
2) this doesn't look the same as what you posted internally with your screen recording. did you back out some stuff or is it not loading right on my side?

<img width="918" alt="Screen Shot 2020-04-01 at 10 13 43 AM" src="https://user-images.githubusercontent.com/49954156/78147482-91e95a00-7401-11ea-936a-a1f4363c18d5.png">

Reply:
Yeah you are definitely missing some files. But they are all in the branch - I just did a clean checkout.  Did you hard refresh your page?

Reply:
Yeah that's definitely not right.  Have you hard refreshed?